pcstac/tests/resources/test_item.py

```diff
@@ -562,7 +562,7 @@ async def test_pagination_token_idempotent(app_client):
 
     # Get the next link
     page_data = page.json()
-    next_link = list(filter(lambda l: l["rel"] == "next", page_data["links"]))
+    next_link = [link for link in page_data["links"] if link["rel"] == "next"]
```

Comment:
Really interesting. This definitely makes it more readable, but does it also increase efficiency?